.github/workflows/container.yaml

```diff
@@ -40,6 +40,7 @@ jobs:
         with:
           images: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}/${{ matrix.variant }}
           tags: |
+            "type=raw,value=latest,enable=${{ github.ref == format('refs/heads/{0}', github.event.repository.default_branch) }}"
```

Comment:
latest gets generated by event=tag by default; i've not bothered with an explicit latest tag on purpose